streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -692,10 +697,4 @@ boolean commitNeeded() {
     RecordCollector recordCollector() {
         return recordCollector;
     }
-
-    // visible for testing only
-    RecordCollector createRecordCollector(final LogContext logContext,
```

Comment:
This existed only so a test could override it. Instead, I added a constructor arg for the test to pass and removed this method.

Reply:
Sounds good!